hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/src/main/java/com/google/cloud/bigtable/hbase2_x/replication/HbaseToCloudBigtableReplicationEndpoint.java

```diff
@@ -66,6 +145,11 @@ public boolean replicate(ReplicateContext replicateContext) {
     return cloudBigtableReplicator.replicate(walEntriesByTable);
   }
 
+  @Override
+  public boolean isStarting() {
+    return false;
```

Comment:
Where is this used and why can we hardcode it to false? Do we start instantaneously?

Reply:
Replaced.